precompiles/assets-erc20/src/lib.rs

```diff
@@ -80,12 +80,12 @@ pub enum Action {
 	Burn = "burn(address,uint256)",
 	Freeze = "freeze(address)",
 	Thaw = "thaw(address)",
-	FreezeAsset = "freeze_asset()",
-	ThawAsset = "thaw_asset()",
-	TransferOwnership = "transfer_ownership(address)",
-	SetTeam = "set_team(address,address,address)",
-	SetMetadata = "set_metadata(string,string,uint8)",
-	ClearMetadata = "clear_metadata()",
+	FreezeAsset = "freezeAsset()",
```

Comment:
This is a breaking change no?

Reply:
Yes, the change in case changes the keccak hash

Reply:
Yes, we need a way to support both to start with